howto-gallery/package.json

```diff
@@ -40,4 +40,4 @@
       "@minecraft/server": "$@minecraft/server"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
I noticed when pulling down this repo and then running `npm run lint` that most (all?) files failed linting due to file endings. I ran `npm run lint -- --fix` and it fixed them all locally but git didn't recognize the difference. Is there something weird going on with line endings?

Reply:
+1, not sure if we are missing some config.

Reply:
This repository doesn't have build pipelines, and this package is used more for snippets, so it's possibly that simply this is just stale.

Reply:
I created this user story to fix the linting issues: [minecraft-scripting-samples fails linting on a fresh clone](https://dev.azure.com/dev-mc/Minecraft/_workitems/edit/1207082)